content/messaging/agent-chat-api/v3.2/index.mdx

```diff
@@ -2800,6 +2800,119 @@ https://api.livechatinc.com/v3.2/agent/action/untag_chat_thread \
 <Section>
 <Text>
 
+### Get Customer
```

Comment:
Please, also remember about adding the new methods to the **Available methods** table in both Web and RTM :)

Reply:
+ Update the changelog with the following info, please:
There's a new method **Get Customer** ([Web](/messaging/agent-chat-api/v3.2/#get-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-customer)) that returns info about a given Customer.